src/vs/workbench/services/hover/browser/hoverWidget.ts

```diff
@@ -443,6 +434,22 @@ export class HoverWidget extends Widget {
 		}
 	}
 
+	private adjustHoverMaxHeight(target: TargetRect): void {
+		let maxHeight = 250;
```

Comment:
This is solving a different tracked issue, right? I think it should be out of scope of this PR as it will need more discussion, in particular I always thought that we would need to integrate a `DomScrollableElement`.

Reply:
This hoverWidget contains a base hoverWidget which has such an element. It's just that this hoverWidget clears out the maxHeight at https://github.com/microsoft/vscode/blob/main/src/vs/workbench/services/hover/browser/hoverWidget.ts#L231

Reply:
That pixel high might cause problems in some scenarios though, and it would need to be tested to make sure the hover correctly flips when real estate isn't there